plugins/delpoyment_stats/app/models/deployment_stats.rb

```
@@ -0,0 +1,63 @@
+require 'sqlite3'
+
+class DeploymentStats
+  @@db = nil
+  PASSWORD = "admin123"
```

rubocop (1.68.0)

[convention] Freeze mutable objects assigned to constants.

(Style/MutableConstant)